service/validator.go

```diff
@@ -27,20 +27,22 @@ type Validator interface {
 
 // ValidatorConfig ...
 type ValidatorConfig struct {
-	jwtValidator       JWTValidator
-	baseURL            string
-	realm              string
-	keyCacher          auth0.KeyCacher
-	signatureAlgorithm jose.SignatureAlgorithm
+	jwtValidator         JWTValidator
+	baseURL              string
+	realm                string
+	keyCacher            auth0.KeyCacher
+	signatureAlgorithm   jose.SignatureAlgorithm
+	externalErrorHandler ExternalErrorHandler
```

Comment:
Isn't it internal error? 🤔

Reply:
I renamed it to `InternalErrorHandler` but I'm up for suggestions.